firstvoices/backend/models/import_jobs.py

```diff
@@ -83,7 +83,7 @@ class Meta:
             "delete": predicates.is_at_least_editor_or_super,
         }
 
-    description = models.CharField(blank=True, max_length=MAX_DESCRIPTION_LENGTH)
```

Comment:
I though this field was supposed to be a `description` that is added to the model if a task fails or is cancelled. Has the purpose of this field changed?